modules/sso/variables.tf

```
@@ -1,11 +1,55 @@
 variable "managed_permission_sets" {
-  type        = list(any)
+  type = list(object({
+    name              = string
+    description       = string
+    attached_policies = list(string)
+    session_duration  = optional(string)
```

```suggestion
    session_duration  = optional(string, "PT12H")
```

Declaring this argument without a default is overriding the try statement in the resource block, resulting in all session durations being changed back to the default of `PT1H1`